code/_onclick/click.dm

```
@@ -393,4 +393,28 @@ var/global/list/click_catchers
 		var/turf/T = screen_loc2turf(screen_loc, get_turf(usr))
 		if(T)
 			T.Click(location, control, params)
-	. = 1
\ No newline at end of file
+	. = 1
+
+// Suppress the mouse macros
+/client/var/next_mouse_macro_warning
+/mob/proc/LogMouseMacro(verbused, params)
+	if(!client)
+		return
+	if(!client.next_mouse_macro_warning) // Log once
+		log_admin("[key_name(usr)] attempted to use a mouse macro: [verbused] [params]")
+		message_admins("[key_name_admin(usr)] attempted to use a mouse macro: [verbused] [html_encode(params)]")
+	if(client.next_mouse_macro_warning < world.time) // Warn occasionally
+		usr.playsound_to(get_turf(usr), sound('sound/misc/sadtrombone.ogg'), FALSE)
```

Nope. Don't do that. Just fail silently.